config/packages/scheb_2fa.yaml

```diff
@@ -0,0 +1,18 @@
+# See the configuration reference at https://symfony.com/bundles/SchebTwoFactorBundle/6.x/configuration.html
+scheb_two_factor:
+    security_tokens:
+        - Symfony\Component\Security\Core\Authentication\Token\UsernamePasswordToken
+        - Symfony\Component\Security\Http\Authenticator\Token\PostAuthenticationToken
+    google:
```

Comment:
Yes it would work for any module as long as the firewall is configured correctly. For example:

```yml 
firewalls:
  profiles:
    two_factor:
      auth_form_path: 2fa_login
      check_path: 2fa_login_check
      multi_factor: true

access_control:
  - { path: ^/profile/2fa, role: IS_AUTHENTICATED_2FA_IN_PROGRESS }
```